### ISSUE TITLE
Welcome Message Box [2023]

### ISSUE THREAD

**Issue by TauCSGO** [danielkrupinski/Osiris#4109]
Good night
Some time ago there was a popou-style message that welcomed osiris.
Is it possible to activate it again?
I think it was in "interfaces" but apparently it's gone

Thank you

**Comment by Xsintashi** [comment]
yes with createCommandMsgBox func in GameUI Interface. Idk is that interface deleted in recent Osiris or moved
